WalletWasabi/BitcoinP2p/P2pBehavior.cs

```diff
@@ -67,7 +67,7 @@ private async Task ProcessInventoryAsync(Node node, InvPayload invPayload)
 		{
 			if (ProcessInventoryVector(inv, node.RemoteSocketEndpoint))
 			{
-				getDataPayload.Inventory.Add(inv);
+				getDataPayload.Inventory.Add(new InventoryVector(node.AddSupportedOptions(inv.Type), inv.Hash));
```

Comment:
Very old pre-segwit nodes can still return witness-stripped transactions but we don't connect to those nodes.